### ISSUE TITLE
SendEmailAsync's first parameter is ignored in favorof joe@constoso.com

### ISSUE THREAD

**Issue by pkellner** [dotnet/AspNetCore.Docs#20860]
First, I can't get sendgrid working, but probably my fault somehow.
However, is not helpful that the passed in `string email` is not used in newEmailAddress, but instead Joe@contoso.com is.
My expectation is that if I can copy the code, there is some hope it will work as is.

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fe1bea60-d173-f6d2-3559-b147699e295a
* Version Independent ID: 98e7aacb-2a01-d581-9b03-bfef2893163e
* Content: [Account confirmation and password recovery in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/accconfirm?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/security/authentication/accconfirm.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/security/authentication/accconfirm.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@pkellner  you're right, but it's pretty obvious how to change that. I'll review this when I next update the article.

**Comment by Rick-Anderson** [comment]
> First, I can't get sendgrid working, but probably my fault somehow.

Create a console app. Send to multiple email accounts. I can't get it to work with microsoft accounts, but yahoo and gmail work. Also take a look at the SendGrid dashboard to see what email is sent.

**Comment by Rick-Anderson** [comment]
https://docs.microsoft.com/en-us/aspnet/core/security/authentication/accconfirm?view=aspnetcore-5.0&tabs=visual-studio#debug-email

**Comment by pkellner** [comment]
Thanks @Rick-Anderson and @guardrex .  I did get sendgrid working, not sure why it didn't first time and I did not realize that the method has nothing to do with asp.net core, just sendgrid which I do know a lot about (I send about a million emails a year with it).

When I googled for asp.net core and sendng email, I ran into a couple other people also (like me) who were not paying enough attention and also had the issue.  Because the code looks so cut-and-pastable, it's easy to not pay attention and think "of course the code is right".  I know both of you see errors like that as if they had a flashing red light on them, but there are a lot of people like me who just see right through stuff like that without recognizing it as an error until the code doesn't work.  But... certainly not that important to fix.  I miss working with you guys.
